src/soroban/contract_client/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from "./assembled_transaction";
+export * from "./basic_node_signer";
+export * from "./client";
+export * from "./rust_result";
+export * from "./sent_transaction";
+export * from "./types";
+export * from "./utils";
```

Comment:
are we SURE all of these should be exported (e.g. `utils` feels excessive)? at least at the package level. remember that it means we have to commit to **every** **single** name, parameter type/count, method, etc.

Reply:
Eh... I could honestly go either way. `utils` exports:

- `DEFAULT_TIMEOUT`
- `withExponentialBackoff`
- `contractErrorPattern`
- `implementsToString`

I could see all of these being useful, and they are all namespaced under `ContractClient`, so maybe it's ok?

But if we want to keep any in particular we could move the definition elsewhere and only export that.

Reply:
But yeah, if we want to keep the implementation small(er), we can remove `utils`. I think we want to export the rest of it, though! It's all used by TS Bindings.

Reply:
Removed `utils`, kept the rest.